app/views/dashboards/base/_index.haml

```diff
@@ -2,10 +2,7 @@
 = render 'index_title'
 - unless GrdaWarehouse::Config.get(:multi_coc_installation)
   = render 'housed_overall'
-- if @html
```

Comment:
Pretty sure this was to prevent the picker from showing up in PDFs.  (It's totally possible this is handled later, but wanted to check.)

Reply:
This template is only every invoked if `@html` is true, so it was dead code:
https://github.com/greenriver/hmis-warehouse/blob/aaf97a25704742593a51a0134cd1b99e18b04646/app/views/dashboards/base/index.haml#L1-L4